client/src/js/base/__tests__/DropDown.test.js

```diff
@@ -0,0 +1,32 @@
+import { DropDown } from "../DropDown";
+
+describe("<DropDown />", () => {
+    const state = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['state' is assigned a value but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4925540)